demos/code/Blocks/Chapter 5 - Functions.js

```diff
@@ -245,34 +126,31 @@ Blockly.Blocks['user_function'] = {
 		this.funcProp_[2] = "";
 		this.funcProp_[3] = this.getVar_;
 		this.funcProp_[4] = false;
-		
+
 		//Allocate function parameters
-		let ptr = this.getInputTargetBlock('valinp1');
-		
 		this.funcParam_ = [];
-		//Loop through the parameters
-		while(ptr){
+		let inputBlock = this.getInputTargetBlock('valueInput');
+
+		while(inputBlock){
```

Comment:
Changed the "ptr" object name to "inputBlock". This is a more accurate description of what this object is and I kept reading "ptr" as "pointer" although I think this was meant to mean "parent"